apps/src/redux/runState.js

```diff
@@ -1,20 +1,20 @@
 /**
  * This duck module tracks whether the current app's run state.
  */
-var _ = require('../lodash');
+import _ from 'lodash';
```

Comment:
You know, it'd be a lot harder to make this mistake if we just used stock lodash everywhere :wink: